docs/release-strategy.md

```diff
@@ -29,10 +29,16 @@ PRs and Issues associated with the next two milestones will be prioritized for r
 
 ## Git Branches and Tags
 
-Every `X.Y` release stream gets a new branch.
+Every `X.Y` release stream gets a new branch `release-X.Y`. The release branch is created with first beta, release candidate, or final `X.Y.0` release.
```

Comment:
Do we need alphas, betas, and RCs? We want an alpha today to avoid breaking community users while we test some downstream build machinery. The pre-release type doesn't really matter for that, so we could just always use alphas. Maybe there's a reason to use different types upstream, though?

Reply:
I would like to start using RCs from here on out with a couple of days of testing between rc1 and final.

- alphas are cut from the main branch and can have known bugs
- betas are cut from a release branch and should have stable API and requirements
- rcs should be stable. When everything goes right, then `1.2.3rc1` tag will become the `1.2.3` tag without any modifications.

Reply:
OK, that makes sense.